ra-optimisation/closed-optimisation-rao/src/main/java/com/farao_community/farao/closed_optimisation_rao/pre_processors/FaraoVariantsPool.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.closed_optimisation_rao.pre_processors;
+
+import com.powsybl.iidm.network.Network;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.*;
+
+/**
+ * @author Sebastien Murgey {@literal <sebastien.murgey at rte-france.com>}
+ */
+public class FaraoVariantsPool {
```

Comment:
Add a delete method which cleans the created network variants ?

Reply:
done